DataManagementSystem/DB/FileCatalogComponents/FileMetadata.py

```diff
@@ -194,13 +205,47 @@ def __getFileID( self, path ):
 
   def __setFileMetaParameter( self, fileID, metaName, metaValue, credDict ):
     """ Set an meta parameter - metadata which is not used in the the data
-        search operations
+        search operations.
+        Modified by JM: make metaName Dirac group aware.
     """
     result = self.db._insert( 'FC_FileMeta',
                           ['FileID', 'MetaKey', 'MetaValue'],
-                          [fileID, metaName, str( metaValue )] )
+                          [fileID, self.getMetaName(metaName, credDict), str( metaValue )] )
     return result
 
+  @staticmethod
+  def fqMetaName(metaName, credDict):
+    return metaName+'_'+credDict['group']
```

Comment:
For the correct format please see pep8 rules (that we adopt with the exclusion of the spacing). Also travis complains (https://travis-ci.org/DIRACGrid/DIRAC/jobs/540768734)

Reply:
The method is obsolete. Removed.